I increased `max-examples` to 100 and there's one more test failure:
```
FAILED array_api_tests/test_operators_and_elementwise_functions.py::test_clip - AssertionError: out.dtype=uint32, but should be uint16 [clip(uint16)]
Falsifying example: test_clip(
    x=array(0, dtype=uint16),
    data=data(...),
)
Draw 1 (min.shape, max.shape): ((), ())
Draw 2 (min): 0
Draw 3 (max): array(0, dtype=uint32)
Draw 4 (kwargs): {'min': 0, 'max': array(0, dtype=uint32)}
```
And I wonder if it's more an issue on the `array-api-tests` side. The standard says:
> If `x` and either `min` or `max` have different data type kinds (e.g., integer versus floating-point), behavior is unspecified and thus implementation-dependent.

And the failing example is:
```python
In [30]: x = np.array(0, dtype=np.uint16)

In [31]: np.clip(x, min=0, max=np.uint32(0))
Out[31]: np.uint32(0)
```
I think promotion here is intentional and if mixed data types of `x` and `min`/`max` is implementation-dependent then `array-api-tests` shouldn't require `x.dtype == np.clip(x, ...).dtype` here. WDYT?

--- a/tools/ci/array-api-xfails.txt
+++ b/tools/ci/array-api-xfails.txt
@@ -18,3 +18,6 @@ array_api_tests/test_sorting_functions.py::test_sort
 # ufuncs signature on linux is always <Signature (*args, **kwargs)>
 # np.vecdot is the only ufunc with a keyword argument which causes a failure
 array_api_tests/test_signatures.py::test_func_signature[vecdot]
+
+# input is cast to min/max's dtype if they're different
+array_api_tests/test_operators_and_elementwise_functions.py::test_clip

